### ISSUE TITLE
Feature request: ignore everything between starting and closing comment

### ISSUE THREAD

**Issue by benface** [FullHuman/purgecss#66]
The current whitelisting features are great, but I think the "CSS comment" one could be improved. Instead of just ignoring one selector/ruleset at a time, I think it should ignore everything between two comments, like this:

```
/* purgecss start ignore */
h1 {
    color: blue;
}
h2 {
    color: red;
}
/* purgecss end ignore */
```

It would make it much easier to ignore large parts of the CSS (even whole files) that you are 100% sure you want to keep even if PurgeCSS can't find the mentioned classes/elements in the markup, because they are added dynamically with JavaScript for instance.

**Comment by jsnanigans** [comment]
Great Idea! thanks

**Comment by gisu** [comment]
👍

**Comment by Ffloriel** [comment]
This feature has been added with the released version 1.0.0.
I'll now close this issue, let me know if something does not work as expected.
